control-plane/roles/gardener/defaults/main/gardener.yaml

```diff
@@ -101,6 +101,7 @@ gardener_shooted_seeds: []
 #   verticalPodAutoscaler: false
 #   pod_cidr:
 #   service_cidr:
+#   high_availability_control_plane: node
```

Comment:
Maybe nice to have some explanation what types are possible (node|zone)

Reply:
Done.